src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/GoogleTagManagerCategoryConnectorDependencyProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManagerCategoryConnector;
+
+use Spryker\Yves\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Yves\Kernel\Container;
+
+class GoogleTagManagerCategoryConnectorDependencyProvider extends AbstractBundleDependencyProvider
+{
+    public const PLUGINS_GOOGLE_TAG_MANAGER_CATEGORY_PRODUCT = 'PLUGINS_GOOGLE_TAG_MANAGER_CATEGORY_PRODUCT';
+
+    public function provideDependencies(Container $container): Container
```

Comment:
missing

Reply:
thx!

strange, i already run phpcbf. No fixable errors found but doc-block is still missing.